Description: Fix requirements.txt
 Without this patch, we get:
  dpkg-gencontrol: warning: can't parse dependency python-cryptography (!= 1.3.0)
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2016-06-08

--- nova-14.0.0~b1.orig/requirements.txt
+++ nova-14.0.0~b1/requirements.txt
@@ -10,9 +10,8 @@ eventlet!=0.18.3,>=0.18.2 # MIT
 Jinja2>=2.8 # BSD License (3 clause)
 keystonemiddleware!=4.1.0,!=4.5.0,>=4.0.0 # Apache-2.0
 lxml>=2.3 # BSD
-Routes!=2.0,!=2.1,!=2.3.0,>=1.12.3;python_version=='2.7' # MIT
-Routes!=2.0,!=2.3.0,>=1.12.3;python_version!='2.7' # MIT
-cryptography!=1.3.0,>=1.0 # BSD/Apache-2.0
+Routes
+cryptography
 WebOb>=1.2.3 # MIT
 greenlet>=0.3.2 # MIT
 PasteDeploy>=1.5.0 # MIT
